.docker/deployment/Dockerfile

```diff
@@ -4,13 +4,15 @@
 #  -e CANISTERIUM_CANISTER_ID="rrkah-fqaaa-aaaaa-aaaaq-cai" \
 #  -e PAT=YOUR_PERSONAL_AT \
 #  fleek/f3o-fe
-FROM fleek/f3o:latest
+FROM fleek/f3o:0.2.2
```

Comment:
consider have a consistent name with the project

Reply:
also the `deployment` directory name doesn't seem to match with context

Reply:
I'ts been renamed to fleek/dfxrust image